### PR TITLE
[21.01] Match environment variable set up by doc building workflow

### LINKED ISSUE #11645: release docs show incorrect warning
This document shouldn't have this notification.

January_2021_Galaxy_Release__v_21_01__—_Galaxy_Project_21_01_documentation

### PR DESCRIPTION
**Pull request by mvdbeek**
Fixes https://github.com/galaxyproject/galaxy/issues/11645

### CODE REVIEW

**Review by nsoranzo — APPROVED** [review]

**Comment by nsoranzo** [comment]
Can you maybe cherry-pick https://github.com/galaxyproject/galaxy/pull/11629/commits/2f0a0249154f2e50fb1b20fabb8d33ad91b09d3f ?

File: lib/galaxy/dependencies/pipfiles/default/Pipfile [hunk 1]
```diff
@@ -87,7 +87,7 @@ cloudbridge = "*"
 social-auth-core = {version = "==3.3.0", extras = ['openidconnect']}
 cloudauthz = "==0.6.0"
 gxformat2 = "*"
-refgenconf = ">=0.7.0"
+refgenconf = ">=0.7.0, <0.10.0"  # https://github.com/galaxyproject/galaxy/issues/11601
```
Comment (nsoranzo) [inline lib/galaxy/dependencies/pipfiles/default/Pipfile:90]: :+1:

**Review by nsoranzo — APPROVED** [review]

**Comment by nsoranzo** [comment]
Should we fix the mypy error by using `VERSION_MINOR = ""` in `lib/galaxy/version.py` ?

**Comment by mvdbeek** [comment]
Let's merge this so we get the warning out of the doc builds, I can open another PR for the lint fix.

File: lib/galaxy/dependencies/pipfiles/default/Pipfile (hunk 1 above)
Comment (nsoranzo) [reply lib/galaxy/dependencies/pipfiles/default/Pipfile:90]: Can you please do the same also in `packages/app/requirements.txt` ?

Comment (mvdbeek) [reply lib/galaxy/dependencies/pipfiles/default/Pipfile:90]: is that the same syntax ?

Comment (nsoranzo) [reply lib/galaxy/dependencies/pipfiles/default/Pipfile:90]: I mean, as done below for sqlalchemy.

**Review by nsoranzo — APPROVED** [review]

File: lib/galaxy/dependencies/pipfiles/default/Pipfile [hunk 2]
```diff
@@ -39,7 +39,7 @@ Markdown = "*"
 # Optional requirement for markdown export.
 # Weasyprint = "==0.42"
 PyYAML = "*"
-SQLAlchemy = "*"
+SQLAlchemy = ">=1.3.22, <1.4.0"  # https://github.com/kvesteri/sqlalchemy-utils/issues/474
```
Comment (nsoranzo) [inline lib/galaxy/dependencies/pipfiles/default/Pipfile:42]: This should have been 1.3.20 (the version we are currently pinning in `requirements.txt` for 21.01) , but not a big deal, I don't think we are going to use pipenv to update deps in the release branch.

File: lib/galaxy/version.py [hunk 3]
```diff
@@ -1,3 +1,3 @@
 VERSION_MAJOR = "21.01"
-VERSION_MINOR = None
+VERSION_MINOR = ""
```
Comment (natefoo) [inline lib/galaxy/version.py:2]: Some of the release tooling expects `None` for the `.0` release, so this change breaks that a bit. However we don't currently make point releases and I am overhauling the process, so I can incorporate this change if necessary.

Comment (nsoranzo) [reply lib/galaxy/version.py:2]: Yes, please, it should be possible to replace `foo is None` with `not foo` or similarly, I suppose.

Comment (natefoo) [reply lib/galaxy/version.py:2]: I assume I should also make it so that when we create future .0 releases, we use the empty string instead of `None`?

Comment (nsoranzo) [reply lib/galaxy/version.py:2]: Yep, we need to keep mypy happy :)

**Pull request merged**